### PR TITLE
Feature/top bar pixels

### PR DESCRIPTION
**Pull request by frapperino**
Working on all window sizes and is consistent with hedvig.com navbar

### CODE REVIEW

File: src/components/TopBar.tsx [hunk 1]
```diff
@@ -135,6 +161,13 @@ export const TopBar: React.SFC<Props> = (props) => (
             ))
           : null}
       </BarProgressContainer>
+      <CollapsedProgressContainer>
```
Comment (palmfjord) [inline src/components/TopBar.tsx]: maybe we should actually hide the whole container if we don't have a progress?

Comment (frapperino) [reply src/components/TopBar.tsx]: hmm, think the naming might be confusing, the CollapsedProgressContainer is only showing the current step in the process

Comment (frapperino) [reply src/components/TopBar.tsx]: Oh I see what you mean!

File: src/components/TopBar.tsx [hunk 2]
```diff
@@ -116,25 +139,32 @@ export const TopBar: React.SFC<Props> = (props) => (
   <Container>
     <Bar>
       <BarContainer>
-        <Logo src="/assets/offering/logo.png" />
+        <EscapeLink href="/hedvig">
```
Comment (palmfjord) [inline src/components/TopBar.tsx]: should this be a link to `https://hedvig.com` rather?

File: src/components/buttons.tsx [hunk 3]
```diff
@@ -36,3 +36,7 @@ export const Button = styled('button')(
     },
   }),
 )
+
+export const RefreshButton = styled('button')({})
```
Comment (palmfjord) [inline src/components/buttons.tsx]: why are these at the bottom or and why are they exported? :p

File: src/components/TopBar.tsx (hunk 2 above)
Comment (frapperino) [reply src/components/TopBar.tsx]: I'm also leaning towards that, but I guess it depends on what makes most sense in the context of web onboarding

File: src/components/buttons.tsx (hunk 3 above)
Comment (frapperino) [reply src/components/buttons.tsx]: forgot to remove those, thx!

File: src/components/TopBar.tsx [hunk 4]
```diff
@@ -65,36 +72,53 @@ const ProgressLabel = styled('div')({
...
+    flexDirection: 'column',
+    alignItems: 'center',
   },
 })
 
-const BarProgressContainer = styled('div')({
-  display: 'flex',
-  width: '60%',
+const CollapsedProgressWrapper = styled('div')({
+  display: 'none',
```
Comment (palmfjord) [inline src/components/TopBar.tsx]: can we remove all these props if it's not gonna be visible anyways?

**Pull request closed without merging**